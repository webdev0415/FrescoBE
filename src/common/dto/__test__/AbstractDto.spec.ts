import {AbstractDto} from '../AbstractDto';

describe('AbstractDto', () => {
    it('AbstractDto snapshot ', () => {
        expect(AbstractDto).toMatchSnapshot();
    });
});
