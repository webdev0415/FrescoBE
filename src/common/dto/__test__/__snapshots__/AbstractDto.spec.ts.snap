// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AbstractDto AbstractDto snapshot  1`] = `[Function]`;
